libr/util/str.c

```diff
@@ -3573,3 +3574,24 @@ R_API char *r_str_scale(const char *s, int w, int h) {
 	free (str);
 	return r_str_list_join (out, "\n");
 }
+
+
+
+R_API int r_charset_encode_str(ut8 *asciistr, ut8 *in, int len_input, charset *custom_charset, int len_charset)
```

Comment:
open brace at right.
use size_t for length

dont put spaces before ++

Reply:
@radare do you mean "r_charset_encode_str ("  instead? I do not find similar cases on the functions definition of the same file.